tests/jetty-jmh/pom.xml

```diff
@@ -57,18 +57,6 @@
         </executions>
       </plugin>
     </plugins>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <groupId>org.apache.maven.plugins</groupId>
-          <artifactId>maven-deploy-plugin</artifactId>
-          <configuration>
-            <!-- No point deploying jmh project -->
-            <skip>true</skip>
```

Comment:
Needs the property defined? Or is that covered by the toplevel tests/pom.xml properties?

Reply:
correct properties are inherited.